daemon/daemon_solaris.go

```diff
@@ -301,9 +301,8 @@ func verifyPlatformContainerSettings(daemon *Daemon, hostConfig *containertypes.
 	return warnings, nil
 }
 
-// platformReload updates configuration with platform specific options
-func (daemon *Daemon) platformReload(config *Config) map[string]string {
-	return map[string]string{}
+// reloadPlatform updates configuration with platform specific options
+func (daemon *Daemon) reloadPlatform(config *Config, attributes map[string]string) {
```

Comment:
Here is my thought.
1. everything reload func has a name with a prefix `reload` as `reloadXxxx`, so I changed the function name
2. In my implementation, it needs to modify `attributes`, then it takes one more parameter, but with no return.